README.rst

```diff
@@ -112,3 +112,7 @@ be sure to run the ``openstack-images.yml`` playbook *before* running the
 ``openstack-container-clusters.yml`` playbook, otherwise the Magnum API will return
 an error referencing an invalid cluster type with image ``None``. This is handled
 automatically if running the full ``openstack.yml`` playbook.
+
+Note that these templates are a tested set against the specific Azimuth
+release. As such, you should make sure to update your CAPI management cluster
+(aka Azimuth) to the latest release before updating to the latest templates.
```

Comment:
It's not always Azimuth and we should not ,,recognize'' CAPI mgmt cluster as Azimuth deployment...

Reply:
Ok, I dropped that bit.